### PR TITLE
Initialize django rest swagger recipe

### PR DESCRIPTION
**Pull request by lsetiawan**
## Overview

This PR adds `django-rest-swagger` package to conda-forge. 
> An API documentation generator for Swagger UI and Django REST Framework.

This is the latest version `2.2.0` compared to #5143, that never got merged.

@conda-forge/staged-recipes Please review and merge this recipe, if everything looks good. Thank you very much!

### CODE REVIEW

**Comment by lsetiawan** [comment]
@ocefpaf Could you help me out with this? Seems like some of the dependencies aren't in conda-forge! :disappointed:

**Comment by scopatz** [comment]
LGTM, but waiting for Circle

**Comment by scopatz** [comment]
@lsetiawan - feel free to add the dependencies in other PRs!

**Comment by lsetiawan** [comment]
Thanks @scopatz for the tips and reviewing so quickly.

**Comment by ocefpaf** [comment]
> @ocefpaf Could you help me out with this? Seems like some of the dependencies aren't in conda-forge!

We need to identify which ones and add them. Let me check...

**Comment by lsetiawan** [comment]
Thanks! @ocefpaf :smile:

File: recipes/django-rest-swagger/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python-coreapi
+    - djangorestframework >=3.5.4
+    - openapi-codec >=1.3.1
```
Comment (ocefpaf) [inline recipes/django-rest-swagger/meta.yaml]: @lsetiawan looks like only this one is missing, right?

Comment (lsetiawan) [reply recipes/django-rest-swagger/meta.yaml]: [`openapi-codec`](https://anaconda.org/search?q=openapi-codec) is too old and it's only for osx. I guess I need to create recipe for that. Alright. Thanks for helping me! :smile:

**Comment by ocefpaf** [comment]
Just merged https://github.com/conda-forge/staged-recipes/pull/7987, as soon as the package is created I'll restart it here.

**Comment by lsetiawan** [comment]
Appreciate you merging this! I see it in anaconda cloud now. :+1:

**Pull request merged**